Classes/common/HTTP/CDTURLSessionInterceptorContext.m

```diff
@@ -0,0 +1,29 @@
+//
+//  CDTURLSEssionFilterContext.m
+//  
+//
+//  Created by Rhys Short on 17/08/2015.
+//
+//
+
+#import "CDTURLSEssionInterceptorContext.h"
+
+@implementation CDTURLSessionInterceptorContext
+
+-(instancetype) init {
```

Comment:
I did attempt it before, but it generates warnings,so I think the best option is to just return nil.